### ISSUE TITLE
DNG generated with FreeDCam can't be opened with Darktable

### ISSUE THREAD

**Issue by sdever** [troop/FreeDcam#104]
Hello,

When trying to open a DNG created by FreeDCam (using rawtodng option in viewer), I get this message in terminal:

```
[rawspeed] DNG Decoder: Unsupported format, uncompressed with no strips.
[temperature] failed to read camera white balance information from `2016_08_06_10_48_02.dng'!
[temperature] failed to read camera white balance information from `2016_08_06_10_48_02.dng'!
allocation failed???
```

However, the file can be opened with Rawtherapee or UFRaw, but I have personal preference for darktable.

**Comment by troop** [comment]
Hm thats confusing. As far i know darktable use libraw. Thats also used by freedviewer rawtherapee and ufraw

**Comment by sdever** [comment]
It seems darktable use rawspeed here instead of libraw to process DNGs.

**Comment by troop** [comment]
i played a bit with hdr merge wich also use rawspeed. it read all my dngs produced by freedcam without problems. also the dng validator says that the dngs are ok. i dont think that this is a problem from freedcam

**Comment by troop** [comment]
could be that it is fixed with that commit 99348eb907e1973d9808b9879e4bc566fcb919b0
